### PR TITLE
adding tZq cards for 13 TeV

### PR DESCRIPTION
**Pull request by maierbenedikt**
... Those are tZq cards for the 3-leptonic channel

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/tzq_4f_NLO/tzq_4f_NLO_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+#***********************************************************************
+# Cuts on the charged leptons (e+, e-, mu+, mu-, tau+ and tau-)        *
+# (more specific gen cuts can be specified in SubProcesses/cuts.f)     *
+#***********************************************************************
+   0  = ptl     ! Min lepton transverse momentum
+  -1  = etal    ! Max lepton abs(pseudo-rap) (a value .lt.0 means no cut)
+   0  = drll    ! Min distance between opposite sign lepton pairs
+   0  = drll_sf ! Min distance between opp. sign same-flavor lepton pairs
+   0  = mll     ! Min inv. mass of all opposite sign lepton pairs
+  30  = mll_sf  ! Min inv. mass of all opp. sign same-flavor lepton pairs
```
Comment (bendavid) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/tzq_4f_NLO/tzq_4f_NLO_run_card.dat]: mll>30GeV cut is ok/desired here?

**Comment by maierbenedikt** [comment]
Currently checking with SUS contacts, which I found are responsible for that sample in the first place,a according to the rationalized requests google doc.

**Comment by bendavid** [comment]
Any update on this?

**Comment by maierbenedikt** [comment]
Ahh, sorry, I think this is a very old pull request which is not needed anymore.

The tZq cards have been added in another request (https://github.com/cms-sw/genproductions/commit/a57aee19b8d587108a9188eed6fa5d1e14504279)

splitted into Z->ll and Z-> nunu final states.

Thanks,
Benedikt

**Comment by maierbenedikt** [comment]
But yes, the 30 GeV cut was okay for everyone.

**Pull request closed without merging**